test/integration/dialects/mssql/regressions.test.js

```diff
@@ -82,4 +82,17 @@ if (dialect.match(/^mssql/)) {
         });
     });
   });
+
+  it('sets the varchar(max) length correctly on describeTable', function() {
```

Comment:
Perhaps `dialect === 'mssql'` check will be required, I don't think `VARCHAR(MAX)` is supported by other dialects.

Reply:
these tests are wrapped in

```js
if (dialect.match(/^mssql/)) {
```